packages/components/Notification/src/__tests__/Notification.test.tsx

```diff
@@ -4,6 +4,12 @@ import * as renderer from 'react-test-renderer';
 import { checkRenderConsistency, checkReRender } from '@fluentui-react-native/test-tools';
 
 describe('Notification component tests', () => {
+  beforeAll(() => {
+    jest.mock('@fluentui-react-native/experimental-appearance-additions', () => ({
+      useHorizontalSizeClass: 'regular',
```

Comment:
Just curious, any reason why you chose regular over compact?

Reply:
Not especially - I was testing on iPad and `'regular'` is the default there. Do you or @joannaqu happen to know what the default was in the tests before I started poking things? Looks like it was probably `'compact'` (in which case I'd like to switch it back for safety), though I haven't tested.

Reply:
Well I tested both ways, and it seems to come out the same regardless, so I'm sticking with `regular` for now. Lemme know if you think I should change it.

Reply:
I think it was mostly tested on iPhone - the designs also primarily show examples of notification on an iPhone. Slight preference to use compact but fine leaving it too!